### PR TITLE
Cf pint

### PR DESCRIPTION
**Pull request by jgleissner**
CloudFormation sample with pint lookup

### CODE REVIEW

File: CloudFormation/pint/sles12sp2.templ [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+            "import re",
+	    "",
+            "def handler(event, context):",
+            "  if event['RequestType'] == 'Delete':",
+            "    cfnresponse.send(event, context, cfnresponse.SUCCESS, {})",
+            "  if 'ECS' in event and event['ResourceProperties']['ECS'] == '1':",
+            "    ecs_tag = '-ecs'",
+            "  else:",
+            "    ecs_tag = ''",
+            "  url = 'https://susepubliccloudinfo.suse.com/v1/amazon/%s/images.json' % event['ResourceProperties']['Region']",
```
Comment (rjschwei) [inline CloudFormation/pint/sles12sp2.templ:54]: Please apply the image filter here to reduce the amount of data that needs to be processed by the lambda function:

'https://susepubliccloudinfo.suse.com/v1/amazon/%/images/active.json' % event['ResourceProperties']['Region']

Will produce a list of active images only thus significantly reducing the work that has to be done in the for loop

File: CloudFormation/pint/sles12sp2.templ [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    "AMIInfoFunction": {
+      "Type": "AWS::Lambda::Function",
+      "Properties": {
+        "Code": {
+	  "ZipFile": { "Fn::Join": ["\n", [
+	    "from botocore.vendored import requests",
+	    "import json",
+            "import cfnresponse",
+            "import sys",
+            "import re",
```
Comment (rjschwei) [inline CloudFormation/pint/sles12sp2.templ:45]: There is an indentation issue, starting at "import cfnresponse" there is an additional indent.

**Comment by jgleissner** [comment]
Since nobody seems to object, I'll merge this.

**Review by rjschwei — COMMENTED** [review]
@jgleissner I did comment and wanted you to change the url used to query, did you make the chane or did you miss/ignore the comment? There was also an indentation issue....

File: CloudFormation/pint/sles12sp2.templ (hunk 2 above)
Comment (jgleissner) [reply CloudFormation/pint/sles12sp2.templ:45]: I didn't see your comments before. There was nothing in the pull requests anyway.

This one is the good old tabs vs spaces. I'll convert them into spaces.

File: CloudFormation/pint/sles12sp2.templ (hunk 1 above)
Comment (jgleissner) [reply CloudFormation/pint/sles12sp2.templ:54]: Ah nice, didn't know that there was a URL for active-only images. Thanks!

**Pull request merged**